ui/packages/consul-ui/app/serializers/application.js

```diff
@@ -227,36 +227,38 @@ export default class ApplicationSerializer extends Serializer {
     // create a Set and add version with only major.minor : ex-1.24.6 as 1.24
     let versionSet = new Set();
     payload.forEach(function (item) {
-      if (item.Meta && item.Meta['consul-version'] !== '') {
+      if (item.Meta && item.Meta['consul-version'] && item.Meta['consul-version'] !== '') {
```

Comment:
```suggestion
      if (item.Meta && item.Meta['consul-version']) {
```

Checking for truthiness here should be sufficient :)

Reply:
done